activation/nipost_state.go

```diff
@@ -0,0 +1,80 @@
+package activation
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spacemeshos/go-spacemesh/codec"
+	"github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+const (
+	challengeFilename = "NIPostChallenge"
+	builderFilename   = "NIPostBuilderState"
```

Comment:
please use better filenames (for example nipost_challenge.bin, nipost_builder_state.bin)